internal/provider/indices_datasource.go

```diff
@@ -33,34 +33,32 @@ type allIndicesResourceModel struct {
-	StorageClass                 types.String                  `tfsdk:"storage_class"`
-	InferenceType                types.String                  `tfsdk:"inference_type"`
-	DocsCount                    types.String                  `tfsdk:"docs_count"`
-	StoreSize                    types.String                  `tfsdk:"store_size"`
-	DocsDeleted                  types.String                  `tfsdk:"docs_deleted"`
-	SearchQueryTotal             types.String                  `tfsdk:"search_query_total"`
-	TreatUrlsAndPointersAsImages types.Bool                    `tfsdk:"treat_urls_and_pointers_as_images"`
-	MarqoEndpoint                types.String                  `tfsdk:"marqo_endpoint"`
-	Type                         types.String                  `tfsdk:"type"`
-	VectorNumericType            types.String                  `tfsdk:"vector_numeric_type"`
-	Model                        types.String                  `tfsdk:"model"`
-	ModelProperties              ModelPropertiesModel          `tfsdk:"model_properties"`
-	NormalizeEmbeddings          types.Bool                    `tfsdk:"normalize_embeddings"`
-	TextPreprocessing            TextPreprocessingModel        `tfsdk:"text_preprocessing"`
-	ImagePreprocessing           ImagePreprocessingModel       `tfsdk:"image_preprocessing"`
-	VideoPreprocessing           VideoPreprocessingModelCreate `tfsdk:"video_preprocessing"`
-	AudioPreprocessing           AudioPreprocessingModelCreate `tfsdk:"audio_preprocessing"`
-	AnnParameters                AnnParametersModel            `tfsdk:"ann_parameters"`
-	MarqoVersion                 types.String                  `tfsdk:"marqo_version"`
-	FilterStringMaxLength        types.String                  `tfsdk:"filter_string_max_length"`
+	Created                      types.String            `tfsdk:"created"`
+	IndexName                    types.String            `tfsdk:"index_name"`
+	NumberOfShards               types.String            `tfsdk:"number_of_shards"`
+	NumberOfReplicas             types.String            `tfsdk:"number_of_replicas"`
+	IndexStatus                  types.String            `tfsdk:"index_status"`
+	AllFields                    []AllFieldInput         `tfsdk:"all_fields"`
+	TensorFields                 []string                `tfsdk:"tensor_fields"`
+	NumberOfInferences           types.String            `tfsdk:"number_of_inferences"`
+	StorageClass                 types.String            `tfsdk:"storage_class"`
+	InferenceType                types.String            `tfsdk:"inference_type"`
+	DocsCount                    types.String            `tfsdk:"docs_count"`
+	StoreSize                    types.String            `tfsdk:"store_size"`
+	DocsDeleted                  types.String            `tfsdk:"docs_deleted"`
+	SearchQueryTotal             types.String            `tfsdk:"search_query_total"`
+	TreatUrlsAndPointersAsImages types.Bool              `tfsdk:"treat_urls_and_pointers_as_images"`
```

Comment:
Should there be a `TreatUrlsAndPointersAsMedia`?

Reply:
That's right, added.